.github/workflows/ci.yml

```diff
@@ -25,10 +25,10 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
       - name: Install dependencies
-        run: npm ci --ignore-scripts
+        run: npm ci
+      - name: Build
+        run: npm run build
```

Comment:
The reason behind moving the build step is undocumented.

Reply:
Would adding the following comment work?
```
Having the Build step before the Lint step is intentional because we must generate `*.css.d.ts` files first
```

Reply:
Can't they be generated without running a full build first? Building before linting is not the right sequence of events.

Reply:
I'm not aware of any way of generating these without running a full build currently. I can investigate it but we've already discussed that this workaround might be good for now. Have you changed your mind?

Reply:
OK then, add the comment please.

Reply:
fb71c2375e611ea7dd6e366da756afbb50923a57 looks good.

Reply:
I've added the comment in fb71c2375e611ea7dd6e366da756afbb50923a57.